src/app/store/submitted-transactions/submitted-transactions.hooks.ts

```diff
@@ -0,0 +1,26 @@
+import { useAppDispatch } from '@app/store';
+
+import { submittedTransactionsActions } from './submitted-transactions.actions';
+import { SubmittedTransaction } from './submitted-transactions.slice';
+
+export function useSubmittedTransactionsActions() {
+  const dispatch = useAppDispatch();
+
+  function dispatchNewTransactionSubmitted({ rawTx, txId }: SubmittedTransaction) {
+    dispatch(submittedTransactionsActions.newTransactionSubmitted({ rawTx, txId }));
+  }
+
+  function dispatchTransactionEnteredMempool(txId: string) {
+    dispatch(submittedTransactionsActions.transactionEnteredMempool(txId));
+  }
+
+  function dispatchTransactionReplacedByFee(txId: string) {
+    dispatch(submittedTransactionsActions.transactionReplacedByFee(txId));
+  }
+
+  return {
+    dispatchNewTransactionSubmitted,
+    dispatchTransactionEnteredMempool,
+    dispatchTransactionReplacedByFee,
+  };
```

Comment:
I would omit the dispatch from the names, this is a redux level concept we don't need to expose

Also, I wonder if this could cause a render loop/unnecessary rerender issue. If I'm not mistaken, on each render, these functions are redefined with a new reference, so in `useSubmitTransactionCallback` where you add one of these to the dep array, it'll recreate that `useCallback` on every render.

I'd define a single useMemo and define these within that, similar to 

https://github.com/hirosystems/stacks-wallet-web/blob/3e4210ce4849a19c145489070e12bad66f9e1259/src/app/features/ledger/hooks/use-ledger-navigate.ts#L7-L87